docs/userguide/networking/work-with-networks.md

```diff
@@ -95,6 +95,55 @@ $ docker network create -d overlay
 
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+## Bridge driver options
```

Comment:
We were wondering with @thaJeztah if you should keep this here then :).
@wenchma @mavenugo wdyt ? (I think it's more important to have it in `network create` reference than here :stuck_out_tongue_closed_eyes: )